boards/silabs/radio_boards/xg29_rb4412a/xg29_rb4412a.dts

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) 2025 Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <silabs/xg29/efr32mg29b140f1024im40.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/regulator/silabs_dcdc.h>
+#include "xg29_rb4412a-pinctrl.dtsi"
+
+/ {
+	model = "Silicon Labs xG29-RB4412A";
+	compatible = "silabs,xg29_rb4412a", "silabs,efr32mg29";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,uart-pipe = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,bt-hci = &bt_hci_silabs;
+		zephyr,display = &ls013b7dh03;
+	};
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		sw0 = &button0;
+		sw1 = &button1;
+		watchdog0 = &wdog0;
+		spi-flash0 = &mx25r80;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpiob 0 GPIO_ACTIVE_LOW>;
+			label = "LED 0";
+		};
+		led1: led_1 {
+			gpios = <&gpiod 1 GPIO_ACTIVE_LOW>;
+			label = "LED 1";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
```

Comment:
non consistent spacing of elements in this file

Reply:
Consistently removed spaces between properties and added spaces between child nodes. I couldn't find any explicit guidance on whether child nodes should be spaced away from properties or not:
* Example in Linux DTS style guide has a space: https://docs.kernel.org/6.12/devicetree/bindings/dts-coding-style.html
* Example in Zephyr docs doesn't: https://docs.zephyrproject.org/latest/build/dts/intro-syntax-structure.html#devicetrees-reflect-hardware
* In the Zephyr tree, top-level nodes seem to consistently have space (between chosen/alias/soc etc, as well as between peripherals in soc nodes), but child nodes for buses, partitions etc are often without space